CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 Changes
 =======
 
+# 4.13.0 / 2022-12-30
+
+* [FEATURE] Add support for AlmaLinux and Rocky Linux [#836][] [@bkabrda][]
+* [FEATURE] Add `monitor_unlisted_consumer_groups` and `kafka_consumer_offsets` to kafka_consumer [#834][] [@ganeshkumarsv][]
+* [BUGFIX] Specify package epoch on RH-like platforms to ensure proper upgrade path [#839][] [@bkabrda][]
+* [BUGFIX] Use the correct socket for system probe on Windows [#833][] [@julien-lebot][]
+* [BUGFIX] Add is_jmx by default to jmxfetch checks [#792][] [@albertvaka][]
+
```

Comment:
Links are missing for these entries, they currently appear like this:

<img width="826" alt="image" src="https://user-images.githubusercontent.com/26872252/160873541-b39ac2a4-b5ca-4cb1-90f7-451b0bb003e2.png">

Reply:
Fixed